src/main/java/dev/meyi/bn/utilities/ColorUtils.java

```diff
@@ -6,8 +6,15 @@
 import java.util.Map.Entry;
 import net.minecraft.client.Minecraft;
 import net.minecraft.client.gui.FontRenderer;
+import net.minecraft.client.gui.GuiScreen;
```

Comment:
Can we discard the changes to the ColorUtils file? It doesn't seem to accomplish anything.

Reply:
yes